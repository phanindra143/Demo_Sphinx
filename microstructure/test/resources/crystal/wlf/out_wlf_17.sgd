*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=   101.2398
  !k_ij(1,2)=     1.6114
  !k_ij(2,2)=    98.9263
  !k_ij(1,3)=     1.3113
  !k_ij(2,3)=     0.9584
  !k_ij(3,3)=   101.7375
MP,Kxx,1,  101.23982
MP,Kyy,1,   98.92630
MP,Kzz,1,  101.73747
MP,Kxx,2,   98.92630
MP,Kyy,2,  101.23982
MP,Kzz,2,  101.73747
MP,Kxx,3,   98.92630
MP,Kyy,3,  101.23982
MP,Kzz,3,  101.73747
MP,Kxx,4,  101.23982
MP,Kyy,4,   98.92630
MP,Kzz,4,  101.73747
MP,Kxx,5,  101.23982
MP,Kyy,5,   98.92630
MP,Kzz,5,  101.73747
MP,Kxx,6,   98.92630
MP,Kyy,6,  101.23982
MP,Kzz,6,  101.73747
MP,Kxx,7,   98.92630
MP,Kyy,7,  101.23982
MP,Kzz,7,  101.73747
MP,Kxx,8,  101.23982
MP,Kyy,8,   98.92630
MP,Kzz,8,  101.73747
MP,Kxx,9,  101.73747
MP,Kyy,9,  101.23982
MP,Kzz,9,   98.92630
MP,Kxx,10,  101.23982
MP,Kyy,10,  101.73747
MP,Kzz,10,   98.92630
MP,Kxx,11,  101.23982
MP,Kyy,11,  101.73747
MP,Kzz,11,   98.92630
MP,Kxx,12,  101.73747
MP,Kyy,12,  101.23982
MP,Kzz,12,   98.92630
MP,Kxx,13,  101.73747
MP,Kyy,13,  101.23982
MP,Kzz,13,   98.92630
MP,Kxx,14,  101.23982
MP,Kyy,14,  101.73747
MP,Kzz,14,   98.92630
MP,Kxx,15,  101.23982
MP,Kyy,15,  101.73747
MP,Kzz,15,   98.92630
MP,Kxx,16,  101.73747
MP,Kyy,16,  101.23982
MP,Kzz,16,   98.92630
MP,Kxx,17,   98.92630
MP,Kyy,17,  101.73747
MP,Kzz,17,  101.23982
MP,Kxx,18,  101.73747
MP,Kyy,18,   98.92630
MP,Kzz,18,  101.23982
MP,Kxx,19,  101.73747
MP,Kyy,19,   98.92630
MP,Kzz,19,  101.23982
MP,Kxx,20,   98.92630
MP,Kyy,20,  101.73747
MP,Kzz,20,  101.23982
MP,Kxx,21,   98.92630
MP,Kyy,21,  101.73747
MP,Kzz,21,  101.23982
MP,Kxx,22,  101.73747
MP,Kyy,22,   98.92630
MP,Kzz,22,  101.23982
MP,Kxx,23,  101.73747
MP,Kyy,23,   98.92630
MP,Kzz,23,  101.23982
MP,Kxx,24,   98.92630
MP,Kyy,24,  101.73747
MP,Kzz,24,  101.23982
*endif
