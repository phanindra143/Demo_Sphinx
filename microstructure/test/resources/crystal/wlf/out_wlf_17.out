            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\WLF-50SiC-Poren\9
             3.00000      -1580906.37846         15693.56494        -14274.96397
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000          -438.27169          -118.24851
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000         35926.17108      -1544771.57825        -10453.19287
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000          -576.49364         15625.00000          -105.67790
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000        -12265.06456         -9413.96884      -1589725.49267
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000           -22.20313            12.77786         15625.00000
  /%
